CLDR-17104 Add guide on researching common usage (#5382)
unicode-org/cldr: docs/site/translation/common-usage.md

---
title: 'Researching Common Usage'
---

# Researching Common Usage

CLDR data aligns to the most common usage in the market — not what a language authority prescribes. For most locales this is straightforward, but for languages with limited digital presence demonstrating common usage can be challenging. This guide provides practical methods for finding and documenting evidence to support your contributions.

See [Requesting Changes](/requesting_changes) for general documentation guidelines.

## Where to Find Evidence

### Published and Printed Media

- **Newspapers and magazines** in your language (print or online) are strong evidence. Look at how dates, numbers, currency symbols, and names are formatted.
- **Government publications**, official forms, and legal documents often reflect established conventions.

> - **Textbooks and educational materials** show widely accepted usage, especially for number formats, date formats, and script conventions.

Reviewer: We may also want to include other materials such as packaging, signage and other printed materials.  In the past there were images including of a cereal box for the German Sharp S discussion related to CLDR-17624

### Digital Sources

- **Websites** that serve your language community. News sites, community portals, and local government sites are good sources.
- **Social media** can show how people actually write in everyday contexts. Look at patterns across many users, not just one person's preference.
- **Mobile apps and software** already localized in your language. Check how dates, numbers, and currency are displayed — these reflect existing conventions.
- **Wikipedia** in your language often follows community-agreed conventions for formatting.

> ### Community and Oral Sources

Reviewer 2: Social media is mentioned, but i think the caveat that what's already there may already be based on CLDR needs to be there. For example, it's recommended to use existing translated software, but that may be based on CLDR.

- **Language community organizations** may have style guides or formatting conventions.
- **Discussions with language speakers** from different regions can help identify the most common usage when written sources are limited. Document these conversations when referencing them.
- **Radio and television broadcasts** may use on-screen text or published schedules that show formatting conventions.

## What to Document

> When submitting a change or voting in the Survey Tool, provide as much supporting evidence as possible. Stronger evidence leads to faster resolution.

Reviewer: ```suggestion
When submitting a request for a data change in JIRA or proposing a new value in the Survey Tool, provide as much supporting evidence as possible. Stronger evidence leads to faster resolution.
```

See [Requesting Changes](/requesting_changes) for general documentation guidelines.

## Where to Find Evidence

### Published and Printed Media

- **Newspapers, magazines, and books** in your language (print or online) are strong evidence. Look at how dates, numbers, currency symbols, and names are formatted.
- **Government publications**, official forms, and legal documents often reflect established conventions.
- **Textbooks and educational materials** show widely accepted usage, especially for number formats, date formats, and script conventions.
- **Packaging, signage, and other printed materials** are also useful — product labels, street signs, and store displays often reflect everyday usage. For example, [CLDR-17624](https://unicode-org.atlassian.net/browse/CLDR-17624) used images of cereal box packaging as evidence in a discussion about the German sharp S.
- **Style guides** published by language authorities, news organizations, or governments reflect agreed-upon conventions and carry significant weight.

### Digital Sources

- **Websites** that serve your language community. News sites, community portals, blogs, and local government sites are good sources.

> - **Social media** can show how people actually write in everyday contexts. Look at patterns across many users, not just one person's preference.

Reviewer 3: Should we add a light disclaimer here that social media tends to have a higher degree of misspellings and loanwords that may cause it to be slightly less valuable than professional media?

Reviewer: +1 I think this is worth considering.

> - **Mobile apps and software** already localized in your language. Check how dates, numbers, and currency are displayed — these reflect existing conventions.

Reviewer: Yes, if you add the warning that @macchiati suggested earlier then you could link to the warning about being careful to not use CLDR as evidence of common usage because we want to avoid being self-referring.

---
title: 'Researching Common Usage'
---

# Researching Common Usage

> CLDR data aligns to the most common usage in the market — not necessarily what a language authority prescribes. For most locales this is straightforward, but for languages with limited digital presence or are not standardized demonstrating common usage can be challenging. This guide provides practical methods for finding and documenting evidence to support your contributions.

Reviewer 3: ```suggestion
CLDR data aligns to the most common usage in the market, which may not align with what a language authority prescribes. For most locales this is straightforward, but languages that have a limited digital presence or are not standardized, demonstrating common usage can be challenging. This guide provides practical methods for finding and documenting evidence to support your contributions.
```

See [Requesting Changes](/requesting_changes) for general documentation guidelines.

## Where to Find Evidence

> > **Note:** Avoid using examples that might be generated. For example, the date at the top of an article, or a table of currency values might have been generated by software, and might differ from what someone would type in manually.

Reviewer 3: ```suggestion
> **Note:** Avoid using examples that might be programmatically generated. For example, the date at the top of an article or a table of currency values might have been generated by software (often CLDR), and might differ from what someone would type in manually.
```

### Published and Printed Media

- **Newspapers, magazines, and books** in your language (print or online) are strong evidence. Look at how dates, numbers, currency symbols, and names are formatted.
- **Government publications**, official forms, and legal documents often reflect established conventions.
- **Textbooks and educational materials** show widely accepted usage, especially for number formats, date formats, and script conventions.

> - **Packaging, signage, and other printed materials** are also useful — product labels, street signs, and store displays often reflect everyday usage. For example, [CLDR-17624](https://unicode-org.atlassian.net/browse/CLDR-17624) used images of cereal box packaging as evidence in a discussion about the German sharp S.

Reviewer 3: ```suggestion
- **Packaging, signage, and other printed materials** are also useful — product labels, street signs, and store displays often reflect everyday usage. For example, [CLDR-17624](https://unicode-org.atlassian.net/browse/CLDR-17624) used images of cereal box packaging as evidence in a discussion about the German sharp S (ẞ).
```

CLDR currently does not have the capacity to handle within-country variation using subdivision tags. For example, Valencian is considered a variant of Catalan. The Valencian Community has the tag `esvc` and in BCP-47 the locale code be expressed as `ca-ES-u-sd-esvc` but that isn't currently supported. In the case of Valencian though there is a variant tag registered in IANA `valencia` so CLDR uses `ca-ES-valencia`.

### Variant Subtags

To help differentiate dialects, communities can register their variations as IANA Variant Tags in the [IANA Language Subtag Registry](https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry), allowing them to be expressed in technical systems. Once a variant tag is accepted by IANA, it can be added to CLDR.

> **Note:** Even with variant data in CLDR, the language variant tag extension is not consistently supported across all technical platforms right now — but it's a Catch-22: we need more interest and submissions before more software platforms will support them.

> ### Revisionist Contributions

Reviewer 3: This section seems a bit wordy. I think just the first paragraph is enough, in the interest of keeping this document brief.

Reviewer: Let's just do that in a future PR.

See [Requesting Changes](/requesting_changes) for general documentation guidelines.

## Where to Find Evidence

### Published and Printed Media

- **Newspapers, magazines, and books** in your language (print or online) are strong evidence. Look at how dates, numbers, currency symbols, and names are formatted.
- **Government publications**, official forms, and legal documents often reflect established conventions.
- **Textbooks and educational materials** show widely accepted usage, especially for number formats, date formats, and script conventions.
- **Packaging, signage, and other printed materials** are also useful — product labels, street signs, and store displays often reflect everyday usage. For example, [CLDR-17624](https://unicode-org.atlassian.net/browse/CLDR-17624) used images of cereal box packaging as evidence in a discussion about the German sharp S.
- **Style guides** published by language authorities, news organizations, or governments reflect agreed-upon conventions and carry significant weight.

> 

Author: ohk will shift it to the top and update the PR

> 

Reviewer: I don't see where this got moved to. Can you point me to the current line where you added the warning about being careful so that we don't accidentally use CLDR usage as evidence of common usage?

> 

Reviewer 3: See line 13

Many languages are actively going through a standardization process. For example Romansh in Switzerland has 5 major regions with distinct conventions and deciding a single Standard Romansh was a difficult process. Currently its been unified as "Rumantsch Grischun" but it is not universally accepted as community members favor their home dialects.

### Regional Variants

When it comes to regional variation it's usually straightforward if there are clear differences between countries. For example, "color" for en_US and "colour" for en_GB. However some places may use both versions, like in India / en_IN — prefer the majority usage.

CLDR currently does not have the capacity to handle within-country variation using subdivision tags. For example, Valencian is considered a variant of Catalan. The Valencian Community has the tag `esvc` and in BCP-47 the locale code be expressed as `ca-ES-sd-esvc` but that isn't currently supported. In the case of Valencian though there is a variant tag registered in IANA `valencia` so CLDR uses `ca-ES-valencia`.

> ### Variant Subtags

Reviewer 4: For sure. That's the 3rd paragraph in this section.

---

> title: 'Researching Common Usage'

Reviewer: ```suggestion
title: 'Researching Customary Modern Usage (a.k.a. Common Usage)'
```

---

> # Researching Common Usage

Reviewer: ```suggestion
# Researching Customary Modern Usage (a.k.a Common Usage) {#researching-common-usage}
```

- **Newspapers, magazines, and books** in your language (print or online) are strong evidence. Look at how dates, numbers, currency symbols, and names are formatted.
- **Government publications**, official forms, and legal documents often reflect established conventions.
- **Textbooks and educational materials** show widely accepted usage, especially for number formats, date formats, and script conventions.
- **Packaging, signage, and other printed materials** are also useful — product labels, street signs, and store displays often reflect everyday usage. For example, [CLDR-17624](https://unicode-org.atlassian.net/browse/CLDR-17624) used images of cereal box packaging as evidence in a discussion about the German sharp S.
- **Style guides** published by language authorities, news organizations, or governments reflect agreed-upon conventions and carry significant weight.

### Digital Sources

- **Websites** that serve your language community. News sites, community portals, blogs, and local government sites are good sources.
- **Social media** can show how people actually write in everyday contexts. Look at patterns across many users, not just one person's preference.

> - **Mobile apps and software** already localized in your language. Check how dates, numbers, and currency are displayed — these reflect existing conventions.

Reviewer 2: This was addressed in <https://github.com/unicode-org/cldr/compare/unicode-org:e117036..unicode-org:3e206b5>

## Challenges with Standardization

National languages have often gone through an institutionalized process standardizing the language. Disadvantaged communities have higher dialect and personal variance. While its most pronounced in speech (not applicable to CLDR), it often manifests in spelling and word choice.

Notably, the "standard" form and the "common usage" are not always identical — for example in English "hitherto" technically means "until now" and by dictionary standards used to be considered the standard form. Common usage would use "until now" or "up to this point".

Many languages are actively going through a standardization process. For example Romansh in Switzerland has 5 major regions with distinct conventions and deciding a single Standard Romansh was a difficult process. Currently its been unified as "Rumantsch Grischun" but it is not universally accepted as community members favor their home dialects.

### Regional Variants

> When it comes to regional variation it's usually straightforward if there are clear differences between countries. For example, "color" for en_US and "colour" for en_GB. However some places may use both versions, like in India / en_IN — prefer the majority usage.

Reviewer: I think it's important to explain the goals of CLDR here (majority usage) when usage is split between 2 or more common versions.

> Many languages are actively going through a standardization process. For example Romansh in Switzerland has 5 major regions with distinct conventions and deciding a single Standard Romansh was a difficult process. Currently its been unified as "Rumantsch Grischun" but it is not universally accepted as community members favor their home dialects.

Reviewer 3: ```suggestion
Many languages are actively going through a standardization process. For example Romansh in Switzerland has 5 major regions with distinct conventions and deciding a single Standard Romansh was a difficult process. Currently it has been unified as "Rumantsch Grischun", but it is not universally accepted as community members favor their home dialects.
```

CLDR currently does not have the capacity to handle within-country variation using subdivision tags. For example, Valencian is considered a variant of Catalan. The Valencian Community has the tag `esvc` and in BCP-47 the locale code be expressed as `ca-ES-sd-esvc` but that isn't currently supported. In the case of Valencian though there is a variant tag registered in IANA `valencia` so CLDR uses `ca-ES-valencia`.

### Variant Subtags

> To help differentiate the dialects the Romansch community has registered the various dialects as IANA Variant Tags to the [IANA Language Subtag Registry](https://www.iana.org/assignments/language-subtag-registry/language-subtag-registry), so they can be expressed in technical systems, e.g. `rm_CH_rumgr` for Rumantsch Grischun.

Reviewer: It is important to have consistency in how language names are referred to. Please fix.
